dotnet/src/SemanticKernel.Abstractions/AI/ContentBase.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.AI;
+
+/// <summary>
+/// Base class for all AI non-streaming results
+/// </summary>
+public abstract class ContentBase
+{
+    /// <summary>
+    /// Raw content object reference. (Breaking glass).
```

Comment:
This will not be the inner response all the times. 

Will update it to something similar, like:
`The inner content representation. Use this to bypass the current abstraction.`


i.e. When we convert a `ChatMessageContent` to a `TextContent` (IChatCOmpletion -> ITextCompletion conversion), the `ChatMessageContent` is the `InnerContent` property of a `TextContent` class.